src/agent/onefuzz-supervisor/src/setup.rs

```diff
@@ -129,10 +138,27 @@ async fn create_setup_symlink(setup_dir: &Path, work_unit: &WorkUnit) -> Result<
 async fn create_setup_symlink(setup_dir: &Path, work_unit: &WorkUnit) -> Result<()> {
     use tokio::fs::os::unix::symlink;
 
-    tokio::fs::create_dir_all(work_unit.working_dir()?).await?;
-
-    let task_setup_dir = work_unit.working_dir()?.join("setup");
-    symlink(&setup_dir, &task_setup_dir).await?;
+    let working_dir = work_unit.working_dir()?;
+
+    tokio::fs::create_dir_all(&working_dir)
+        .await
+        .with_context(|| {
+            format!(
+                "unable to create workign directory: {}",
```

Comment:
```suggestion
                "unable to create working directory: {}",
```
Might want to quickly grep for introductions of this typo.